app/views/manage/events/show.html.haml

```diff
@@ -0,0 +1,7 @@
+= render "layouts/manage/page_title", title: "Edit Event"
+
+= render "form"
+.center
+  = link_to 'Delete', manage_event_path(@event), method: :delete, class: 'btn btn-danger' , data: { confirm: 'Are you sure? This action is irreversible.' }
```

Comment:
Can we move this to the top a long with a cancel button to match other forms on the app? 

ex: 
<img width="976" alt="image" src="https://user-images.githubusercontent.com/22222538/100554549-5be7c880-3263-11eb-97c3-f8a935faf436.png">